### DISCUSSION TITLE
how to install a specific version of linkerd using cli ?

### DISCUSSION THREAD

**Discussion by shyam-ks** [linkerd/linkerd2#11318] [General] [answered]
is there way to install specific version of linkerd using linkerd install --version ?

**Comment by alpeb** [comment, marked as answer]
No, for that you would have to download a CLI version specific to the version you want, e.g. for downloading the CLI for 2.12.0:
```
curl -sL https://run.linkerd.io/install | LINKERD2_VERSION=stable-2.12.0 sh
```
OTOH, you can do that directly with helm, but you have to find out first the chart versions corresponding to the linkerd version you want, e.g. for 2.12.0:
```
helm install linkerd-crds -n linkerd --create-namespace --version 1.4.0 linkerd/linkerd-crds
helm install linkerd-control-plane -n linkerd -f certs.yml --version 1.9.0 linkerd/linkerd-control-plane
```
